### DISCUSSION TITLE
Error Encrypting Private Key

### DISCUSSION THREAD

**Discussion by Codeprince001** [Cyfrin/foundry-full-course-cu#3000] [Q&A]
I get an error **Error: Invalid string length** any time i run **cast wallet import your-account-name --interactive**
The private key used was exactly the same as provided in Anvil

![foundry error](https://github.com/user-attachments/assets/6b3b11dc-59b2-4bb0-920a-fe5615ae805e)

**Comment by EngrPips** [comment]
Hello @Codeprince001, I believe you figured it out.

**Reply by Codeprince001** [reply]
Yes

**Reply by EngrPips** [reply]
Awesome, Keep crushing it.
